2-Browsers/Week1/homework/ex4-whatsTheTime/index.js

```diff
@@ -6,9 +6,10 @@
 2. Have the function execute when it's loading in the browser.
 ------------------------------------------------------------------------------*/
 function addCurrentTime() {
-  const date = new Date();
-  const currentTime = date.toLocaleTimeString();
-  document.body.innerHTML = `<div style = font-size:30px>${currentTime} </div>`;
+  setInterval(() => {
```

Comment:
Okay, I implemented it.

Reply:
Does it also make sense to you why?

Reply:
Because we want to run it just once, but what I did first: 
  setInterval(addCurrentTime, 1000);
 window.addEventListener('load', addCurrentTime);
we called the function twice. Right?

Reply:
Yeah correct, and this way we don't put the interval inside the function as the function should just do its thing and not worry about the delay.

Reply:
Okay, thanks.